test/scripts/upgrade-ovn.sh

```diff
@@ -257,13 +269,17 @@ set_default_ovn_manifest_params
 print_ovn_manifest_params
 
 # create upgraded OVN manifests from checked-out branch
-create_ovn_kube_manifests
 set_cluster_cidr_ip_families
+detect_apiserver_url
+create_ovn_kube_manifests
 install_ovn_image
 
 pushd ../dist/yaml
 
-# install updated ovnkube-node daemonset 
+# install updated k8s configuration for ovn-k (useful in case of ClusterRole updates)
+run_kubectl apply -f ovn-setup.yaml
```

Comment:
so this case we will skip creating ovnk namespaces because of `--after-initial-setup` was there an issue if we recreate the namespaces again ? is that related to kube upgrade or this something else ?

Reply:
Right, so skipping the namespace part is not strictly necessary for this PR; the crucial part is to avoid applying  the configmap again. 

I included the namespaces and the ServiceAccount in the part that we set /only/ during the initial setup thinking that these should not change during an upgrade test in order to be on the safe side. I might be wrong about it! :) This only affects the upgrade test anyway.